kolibri/plugins/coach/assets/src/views/plan/LessonContentPreviewPage/index.vue

```diff
@@ -170,6 +171,21 @@
       content() {
         return this.currentContentNode;
       },
+      translatedLicense() {
+        if (
+          licenseTranslations[global.languageCode] &&
```

Comment:
`import { currentLanguage } from 'kolibri.utils.i18n';`
Rather than relying on an unparsed global variable.

Could even abstract this slightly more and create a wrapper function in the `i18n` module that does this.

Reply:
agreed - I was wondering why we were using the global at all actually